extensions/pyRevitTools.extension/checks/CADAUDIT_check.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+
+# Import Reference Modules
+from pyrevit import script, revit, DB, DOCS
+from pyrevit.preflight import PreflightTestCase
+
+from System.Windows import Window # Used for cancel button
+from rpw.ui.forms import (FlexForm, Label, Separator, Button) # RPW
+from rpw.ui.forms.flexform import RpwControlMixin, Controls # Used to create RadioButton
+
+from pyrevit.coreutils import Timer # Used for timing the check
+from datetime import timedelta # Used for timing the check
+
+doc = DOCS.doc
+ac_view = __revit__.ActiveUIDocument.ActiveView # Get the active view
+
+
+# Definitions
+
+def collect_cadinstances(active_view_only):
+    """     Collect ImportInstance class from whole model or from just active view  """
+    #cadinstances = DB.FilteredElementCollector(doc).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
+    if active_view_only:
+        cadinstances = DB.FilteredElementCollector(doc, ac_view.Id).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
+    else:
+        cadinstances = DB.FilteredElementCollector(doc).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
+    
+    if len(cadinstances) >0:
+        return cadinstances, len(cadinstances)
+    else:
+        print("No CAD instances found in the {}.".format("active view" if active_view_only else "model"))
```

Comment:
perhaps it is better to use [forms.alert(exitscript=True)](https://docs.pyrevitlabs.io/reference/pyrevit/forms/#pyrevit.forms.alert) here

![image](https://github.com/user-attachments/assets/29c1dcb4-6074-4b82-aef0-2ee7a33032ff)

Reply:
fixed by #9fd9c57950dc859e12dd2c45bf724827d94fc838